### ISSUE TITLE
Error installing vue-cli because coffeescript moved

### ISSUE THREAD

**Issue by fellz** [vuejs/vue-cli#655]
Hi
System: Windows 10, node version 8.3.0
`npm i -g vue-cli` gives this error:

```
> coffee-script@1.12.8 postinstall C:\Users\Dave\AppData\Roaming\npm\node_modules\vue-cli\node_modules\coffee-script
> node --eval 'if (require("./package.json").name === "coffee-script") { var red, yellow, cyan, reset; red = yellow = cyan = reset = ""; if (!process.env.NODE_DISABLE_COLORS) { red = "\x1b[31m"; yellow = "\x1b[33m"; cyan = "\x1b[36m"; reset = "\x1b[0m"; } console.warn(red + "CoffeeScript has moved!" + reset + " Please update references to " + yellow + "\"coffee-script\"" + reset + " to use " + yellow + "\"coffeescript\"" + reset + " (no hyphen) instead."); console.warn("Also, a new major version has been released under the " + yellow + "coffeescript" + reset + " name on NPM. This new release targets modern JavaScript, with minimal breaking changes. Learn more at " + cyan + "http://coffeescript.org" + reset + "."); console.warn(""); }'

[eval]:1
'if
^^^

SyntaxError: Invalid or unexpected token
    at createScript (vm.js:74:10)
    at Object.runInThisContext (vm.js:116:10)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:573:30)
    at evalScript (bootstrap_node.js:435:27)
    at startup (bootstrap_node.js:139:9)
    at bootstrap_node.js:578:3
```
i guess it's because of this 
https://www.npmjs.com/package/coffee-script
Now coffee-script is coffeescript

**Comment by timfcsm** [comment]
same error with node 6.11.0

**Comment by a062670** [comment]
CoffeeScript has moved!

https://www.npmjs.com/package/coffee-script

**Comment by agogex** [comment]
+1 The same problem. How to fix it?

**Comment by lihua123569** [comment]
how to fix??

**Comment by posva** [comment]
It's something that has to be fixed in a dependency of one of our dependencies. Someone already created a PR for it at https://github.com/jonschlinkert/gray-matter/pull/56

So we'll wait for it to be released and then reinstalling vue-cli would reinstall the updated version of gray-matter

Ideally, we should also ask https://github.com/segmentio/metalsmith/ to update to that version and finally, update the version in the package.json file here

**Comment by posva** [comment]
I'm leaving this open hoping that people will see it instead of opening new issues

**Comment by lihua123569** [comment]
OK, let's try again tomorrow

**Comment by cloudbluedigital** [comment]
oh god I just wanted to start with vue-cli and get the same error haha

**Comment by yyx990803** [comment]
Unfortunately `metalsmith` is using an outdated major version of `gray-matter` so that PR won't actually fix `vue-cli`. Looks like we'll have to fork metalsmith...

**Comment by yyx990803** [comment]
Weird thing though, installation seems to work fine on MacOS despite the warning. Is this Windows only?

**Comment by yyx990803** [comment]
https://github.com/jashkenas/coffeescript/pull/4809

**Comment by yyx990803** [comment]
Should be fixed in 2.9.2 by explicitly pinning `coffee-script` version

**Comment by bigmax01** [comment]
环境变量配好了吗
